index.html

```diff
@@ -1,12 +1,202 @@
+						<a href="#" class="sec-nav__link">Log out</a>
+					</li>
+				</ul>
+				<div class="main-nav flex-row">
+					<a href="index.html" class="logo flex-row">
+						<img src="images/home/biotech.svg" alt="" /><span>Techno&nbsp;</span
+						><img src="images/home/parallelarrows.svg" alt="" />
+					</a>
+					<ul class="main-nav__list">
+						<li class="main-nav__item">
+							<a href="about.html" class="main-nav__link">About</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="index.html" class="main-nav__link" id="home">Home</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Program</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Join</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Sponsor</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">News</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" id="tecno" class="main-nav__link">Techno campaign</a>
+						</li>
+					</ul>
+					<ul class="main-nav__list__mobile">
+						<li class="main-nav__item XIcon">
+							<img
+								src="images/home/XIcon.svg"
+								alt="close"
+								class="main-nav__link"
+							/>
+						</li>
+						<li class="main-nav__item">
+							<a href="about.html" class="main-nav__link">About</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="index.html" class="main-nav__link">Home</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Program</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Join</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">Sponsor</a>
+						</li>
+						<li class="main-nav__item">
+							<a href="#" class="main-nav__link">News</a>
+						</li>
+					</ul>
+				</div>
+				<img
+					src="images/home/hamburgermenu.svg"
+					alt="burgermenu"
+					class="burger"
+				/>
+			</nav>
+		</header>
+		<main class="card">
+			<header class="card__header">
+				<h6 class="card__subtitle">
+					"Hi! Make Money while making the world a better place"
+				</h6>
+				<h1 class="card__title">
+					THE INTERNATIONAL TECHNOPRENEURSHIP <br />
+					SUMMIT 2022
+				</h1>
+			</header>
+			<div class="card__body">
+				<p class="card__text">
+					An opportunity to learn to do the best you can while getting good at
+					your passion and also earning your cool cash, anchored by brilliant
+					individuals from over 90 countries.
+				</p>
+			</div>
+			<div class="card__footer">
+				<p class="date">2022.12.01(THU) ~ 16(FRI)</p>
+				<p class="venue">@ National arts theatre, Iganmu, Lagos, Nigeria</p>
+			</div>
+		</main>
+		<section class="main-program flex-column">
+			<h4 class="main-program__heading">Main<span> Pro</span>gram</h4>
+			<ul class="main-program__items">
```

Comment:
Kindly, I suggest you use -instead of UL/LI- the ARTICLE semantic tag, this is ideal for repetitive content like this